packages/teams-js/src/public/constants.ts

```diff
@@ -49,15 +61,32 @@ export enum HostName {
   teamsModern = 'TeamsModern',
 }
 
-// Ensure these declarations stay in sync with the framework.
+/**
+ * FrameContexts provides information about the context in which the app is running within the host.
+ * Developers can use FrameContexts to determine how their app should behave in different contexts,
+ * and can use the information provided by the context to adapt the app to the user's needs.
+ * 
+ * @example 
+ * If the app is running in the settings context, the developer may want to display options for the 
+ * user to configure their preferences. If the app is running in the content context, the developer 
+ * may want to display information relevant to the content the user is currently viewing. 
+ */
 export enum FrameContexts {
+  /** App's frame context from where settings page can be accessed. */
```

Comment:
>App's frame context from where settings page can be accessed. [](http://example.com/codeflow?start=6&length=61)

Should we add a link to [https://learn.microsoft.com/microsoftteams/platform/tabs/how-to/create-tab-pages/configuration-page?tabs=teamsjs-v2](https://learn.microsoft.com/microsoftteams/platform/tabs/how-to/create-tab-pages/configuration-page?tabs=teamsjs-v2) ?

Reply:
Read through this link and found it to be helpful to include in docs. Will add in latest commit.